changelog.d/6406.bugfix

```diff
@@ -0,0 +1 @@
+Fix bug: parameter missmatch in `_register_email_threepid` function call from `register_user`.
```

Comment:
please could you update this to describe the symptoms of the bug that the user would see?

Reply:
changed